src/util.ts

```diff
@@ -1,7 +1,86 @@
 import { debug, error, info, warning } from "@actions/core";
+import moment from "moment";
 
 import { ActionLogger } from "./github/types";
+import { MonthWithMatch } from "./report/types";
 
 export function generateCoreLogger(): ActionLogger {
   return { info, debug, warn: warning, error };
 }
+
+/**
+ * Extract the matches calculated from a date
+ * @param dates An object that extends { date: string } with it's own custom value
+ * @param getAmount The method to extract such custom value from the object to add to the metrics
+ * @param returnAverage True if the result should be the average for the metrics for a month,
+ * false if it must be the accumulation
+ * @returns Returns an array of objects of type {month: "MM YY", matches: number}
```

Comment:
Looks like it's actually an array of `[string, number]` arrays, not of `{month, matches}` objects.

```suggestion
 * @returns Returns an array of tuples - a month in the "MM YY" format and a corresponding number of matches.
```

Reply:
You are right. I changed the type of value and forgot to update that doc. Good catch!